modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-1_{context}"]
+= Release notes for {gitops-title} 1.1
+
+[id="gitops-features-1-1_{context}"]
+== GitOps features
+
+.GitOps features
+[cols="1,1",options="header"]
+|===
+| {gitops-title} Product | Release
+| Argo CD
+| General availability (GA)
+| Argo CD ApplicationSet
+| Tech Preview (TP)
+| {gitops-title} Application Manager (kam)
+| Tech Preview (TP)
+| {gitops-title} Service
+| Tech Preview (TP)
+|===
+
+
+[id="new-features-1-1_{context}"]
+== New features
+{gitops-title} 1.1 is now available on {product-title} 4.7. {gitops-title} 1.1 is updated to support:
```

Comment:
Remove: {gitops-title} 1.1 is updated to support: 
Also remove bullet points on lines 30-33

Reply:
Done